### ISSUE TITLE
Cannot compile  GNU Hello in outside linux filesystem image

### ISSUE THREAD

**Issue by sunjoong** [Microsoft/BashOnWindows#2025]
* Your Windows build number:  (Type `ver` at a Windows Command Prompt)
Microsoft Windows [Version 10.0.15063]

* What you're doing and what's happening:  (Copy&paste specific commands and their output, or include screen shots)

It's a very simple test. I compiled [GNU Hello](http://ftp.gnu.org/gnu/hello/hello-2.10.tar.gz) in linux filesystem image and outside that. In linux filesystem image, I can compile it, but outside that, I cannot comile, with same condition and command but position.

CASE1: Sucess
```
SUNJOONG-DESKTOP ~ # cd ~
SUNJOONG-DESKTOP ~ # tar xvzf ~/hello-2.10.tar.gz
SUNJOONG-DESKTOP ~ # cd ~/hello-2.10/
SUNJOONG-DESKTOP hello-2.10 # ./configure
SUNJOONG-DESKTOP hello-2.10 # make
```
CASE2: Fail
```
SUNJOONG-DESKTOP ~ # cd /mnt/c/Users/sunjoong/Desktop
SUNJOONG-DESKTOP Desktop # tar xvzf ~/hello-2.10.tar.gz
SUNJOONG-DESKTOP ~ # cd /mnt/c/Users/sunjoong/Desktop/hello-2.10/
SUNJOONG-DESKTOP hello-2.10 # ./configure
SUNJOONG-DESKTOP hello-2.10 # make
```

**Comment by MikeGitb** [comment]
What is the error you get?

**Comment by sunjoong** [comment]
@MikeGitb - In CASE2 only, not in CASE1;
```
SUNJOONG-DESKTOP hello-2.10 # make
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /mnt/c/Users/sunjoong/Desktop/hello-2.10/build-aux/missing aclocal-1.14 -I
 m4
/mnt/c/Users/sunjoong/Desktop/hello-2.10/build-aux/missing: line 81: aclocal-1.14: command not found
WARNING: 'aclocal-1.14' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make: *** [Makefile:1450: aclocal.m4] Error 127
```

**Comment by sunjoong** [comment]
I found why this will happens.

I did not modify `'acinclude.m4' or 'configure.ac' m4 files included by 'configure.ac'`.
But, `make` knew those files were modified because of `VFS design`.
https://github.com/Microsoft/BashOnWindows/issues/1939#issuecomment-294972527

That's worthy to be called `bug` because  it will make many potential problems.
Now, I understand why that happened and  knew @SvenGroot had been self-assigned to this `bug`,
so I'll close this issue.
